lib/ui/home.dart

```diff
@@ -127,7 +129,7 @@ class _HomeState extends State<Home> with WidgetsBindingObserver {
 
   Future<void> _retrieveDynamicLink(HomeViewmodel viewModel) async {
     final PendingDynamicLinkData data =
-    await FirebaseDynamicLinks.instance.retrieveDynamicLink();
+        await FirebaseDynamicLinks.instance.retrieveDynamicLink();
```

Comment:
try to avoid whitespace changes if you can

Reply:
Aaaahw should be a code auto format change... reverting